be/src/Project/Services/Catalog/Catalog.Api/Services/Categories/CategoriesService.cs

```diff
@@ -211,6 +226,25 @@ public async Task<CategoryServiceModel> UpdateAsync(UpdateCategoryServiceModel m
             return Get(new GetCategoryServiceModel { Id = category.Id, Language = model.Language, OrganisationId = model.OrganisationId, Username = model.Username });
         }
 
+        private async Task OrderingCategoriesAsync(int source, int destination)
+        {
+            var categories = _context.Categories.Where(x => x.IsActive);
```

Comment:
**question:** wouldn't it be possible to fetch only categories where order needs to be updated? Taking all categories is very dangerous. Imagine there are hundreds or thousands of them